### DISCUSSION TITLE
Help for css - how to style custom widgets

### DISCUSSION THREAD

**Discussion by JustSimplyKyle** [JakeStanger/ironbar#33] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/68589851/200118517-85430ffa-2f16-473e-98a4-d9ff151b2c56.png)
I currently want to rice the most inner button(e.g. the "restrart" button), but I can't seem to find a way in?

**Comment by JakeStanger** [comment, marked as answer]
This is what the `name` and `class` properties on each widget object are for. In GTK CSS, names work like IDs, and classes work the same way as ever.

You could currently target your restart button with `.power-btn:nth-child(2)` or by adding a name and then something like `#restart`.

I would advise having a read-up or watching some videos on some CSS selector basics. There's not too much to them and they'll get you a long way.

**Reply by JustSimplyKyle** [reply]
I thought they were sub-classes to each other, no wonder I couldn't select them
You're right, I should get some crash-courses on CSS
